tests/test_get_merge_base.py

```diff
@@ -39,7 +39,9 @@ def test_get_merge_base(tmp_path, capsys, checkout, monkeypatch):
 
     shutil.copytree(checkout / "cpython", tmp_path / "cpython")
 
-    get_merge_base.main(True, "linux-x86_64-linux", False, [], tmp_path / "cpython")
+    get_merge_base.main(
+        True, "linux-x86_64-linux", False, False, False, tmp_path / "cpython"
```

Comment:
Doesn't need to happen here, but I think the positional arguments are getting unwieldy enough that keyword-arguments could help here (and in the rest of the diff, below):

Reply:
Yeah, this is kind of a weird one.  This is just a reflection of the positional arguments passed on the commandline from the Github Actions side (where positional arguments are a lot easier to deal with).